src/osdc/Objecter.cc

```diff
@@ -3435,6 +3435,7 @@ void Objecter::handle_osd_op_reply(MOSDOpReply *m)
     // new tid
     s->ops.erase(op->tid);
     op->tid = ++last_tid;
+    s->ops[op->tid] = op; // add op to ops again
```

Comment:
@yuyuyu101 That's PR is right around the time that the RBD fsx tests started to periodically fail